addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -157,16 +157,15 @@
                         </div>
                         <div id="barcode_scanner" class="col-12 col-lg-6 o_setting_box">
                             <div class="o_setting_left_pane">
-                                <field name="barcode_scanner"/>
                             </div>
                             <div class="o_setting_right_pane">
-                                <label for="barcode_scanner" string="Barcode Scanner"/>
+                                <label for="barcode_nomenclature_id" string="Barcode Scanner"/>  
```

Comment:
shouldn't you remove this one? There is already a label `barcode_nomenclature_id` below? (or remove the one below)